File: tests/fixtures/code/sample.py
def greet(name: str):
    print(f"Hello, {name}")


def compute(n: int) -> int:
    total = 0
    for i in range(n):
        if i % 2 == 0:
            total += i
        else:
            total += i * 2
    return total


def main():
    greet("world")
    for i in range(3):
        print("i:", i)
    value = compute(10)
    print("value:", value)


if __name__ == "__main__":
    main()

File: tests/fixtures/code/sample.go
package main

import "fmt"

func greet(name string) {
    fmt.Printf("Hello, %s\n", name)
}

func compute(n int) int {
    total := 0
    for i := 0; i < n; i++ {
        if i%2 == 0 {
            total += i
        } else {
            total += i * 2
        }
    }
    return total
}

func main() {
    greet("world")

    for i := 0; i < 3; i++ {
        fmt.Println("i:", i)
    }

    value := compute(10)
    fmt.Println("value:", value)
}

